### ISSUE TITLE
Mame Current Core for Switch

### ISSUE THREAD

**Issue by Smoker1** [libretro/mame#111]
Request MAME be added to the Switch Build. So far MAME2003 works perfectly regardless of Roms that are perfect for that specific Build. But if the later Cores (2010, 2014, 2015, 2016, or Current) were added, then there would be more fixed Games that would be able to run.

**Comment by SumavisionQ5** [comment]
Can not agree more.

**Comment by tcamargo** [comment]
I tried to compile mame for switch but it need some big changes.Anyway, I would wait for aarch64 jit to be available in mame.

mame 201x versions are not available here.

**Comment by SumavisionQ5** [comment]
> I tried to compile mame for switch but it need some big changes.Anyway, I would wait for aarch64 jit to be available in mame.
> 
> mame 201x versions are not available here.

Batocera has got MAME working on N2 which uses aarch64 linux.
https://github.com/batocera-linux/batocera.linux/tree/master/package/batocera/emulators/mame
Maybe their codes can be borrowed to get it working for Switch, at least without JIT dynarec.

**Comment by tcamargo** [comment]
Yes, batocera-linux supports mame standalone and libretro core on aarch64 for both n2 and vim3. I'm the mame maintainer there too. ;)

Switch problem is the lack of NX (Switch) supports in the many mame libraries.

**Comment by foreversos** [comment]
> Yes, batocera-linux supports mame standalone and libretro core on aarch64 for both n2 and vim3. I'm the mame maintainer there too. ;)
> 
> Switch problem is the lack of NX (Switch) supports in the many mame libraries.

what's highest version mame that  NX can use ?
